lib/cisco_node_utils/vpc.rb

```diff
@@ -62,6 +66,14 @@ def enable
       config_set('vpc', 'feature', state: '')
     end
 
+    def fabricpath_feature
```

Comment:
Since you're only using these helper functions once below, my personal preference would be not to add these and just to call the FabricpathGlobal methods directly:

```ruby
      FabricpathGlobal.fabricpath_feature_set(:enabled) if
        switch_id && :enabled != FabricpathGlobal.fabricpath_feature
```